packages/ssz/test/perf/iterate.test.ts

```diff
@@ -55,16 +54,14 @@ describe("readonly values - iterator vs array", () => {
   });
 });
 
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 function createBalanceList(count: number) {
-  const VALIDATOR_REGISTRY_LIMIT = 1099511627776;
+  const validatorRegistryLimit = 1099511627776;
 
-  const balancesList = new ListBasicType(new UintNumberType(8), VALIDATOR_REGISTRY_LIMIT);
+  const balancesList = new ListBasicType(new UintNumberType(8), validatorRegistryLimit);
   const balancesStruct = Array.from({length: count}, () => 31217089836);
   return balancesList.toViewDU(balancesStruct);
 }
 
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Why is this rule not getting enforced anymore?  Seems like something we want to have still!

Reply:
We have not yet enabled rule `useExplicitFunctionReturnType` because of the limitations of anonymous functions in the object. The fix will come very soon, so we will enable that. 

https://github.com/biomejs/biome/pull/5348

Reply:
looks like that PR was merged.  You rock for contributing for biomejs btw!!!!  Is that feature available now?

Reply:
Not available now, but hopefully will arrive in `1.9.5`

Reply:
Ok.   We should leave a `TODO: ` for the things that cannot be implemented now so there is a note for the future

Reply:
Adding TODO: for adding `biome-ignore` later on seems weird to me. Once we have that rule enabled, it will automatically enforce and let us know which functions don't comply with it and then we either fix those or add ignore.